Formula/aftman.rb

```diff
@@ -0,0 +1,25 @@
+class Aftman < Formula
+  desc "Toolchain manager for Roblox, the prodigal sequel to Foreman"
+  homepage "https://github.com/LPGhatguy/aftman"
+  url "https://github.com/LPGhatguy/aftman/archive/refs/tags/v0.2.7.tar.gz"
+  sha256 "2c4f191bfce631abff185a658b947105055da687c409e09ea80786be4c32b75e"
+  license "MIT"
+  head "https://github.com/LPGhatguy/aftman.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@1.1"
```

Comment:
No linkage with OpenSSL on macOS. We should double-check that this isn't building a vendored OpenSSL.